code/datums/elements/noisy_movement.dm

```diff
@@ -0,0 +1,25 @@
+/datum/element/noisy_movement
+	element_flags = ELEMENT_BESPOKE|ELEMENT_DETACH_ON_HOST_DESTROY
```

Comment:
why does this element have `ELEMENT_DETACH_ON_HOST_DESTROY`, it only clears signals on destroy that happens automatically